src/transformers/models/wav2vec2/tokenization_wav2vec2.py

```diff
@@ -142,6 +142,12 @@ def __init__(
             self.encoder = json.load(vocab_handle)
         self.decoder = {v: k for k, v in self.encoder.items()}
 
+        # make sure that tokens made of several
+        # characters are not split at tokenization
+        for token in self.encoder.keys():
+            if len(token) > 1:
+                self.unique_no_split_tokens.append(token)
```

Comment:
I don't understand this part: `unique_no_split` is intended for use with special tokens only, yet you put every tokens in it here.

Reply:
`Wav2Vec2CTCTokenizer` is supposed to work have vocabularies that consist of characters and/or phonemes, but not subwords. Hence, I assumed that all tokens of the vocabulary consists of only one unicode character. However, there are some exceptions, such as: `"æ̃"` => this phoneme consists of two unicode characters and thus leads to an error as described in the issue. To solve the problem, I put all characters of length > 1 in `self.unque_no_split_tokens`... => is there a better way to handle it here (cc @n1t0)?

Reply:
Thanks for the explanation, it makes sense to me!